recipes-bsp/secure-boot-otp-provisioning/secure-boot-otp-provisioning_0.1.bb

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Su Bao Cheng <baocheng.su@siemens.com>
+#
+# SPDX-License-Identifier: MIT
+#
+
+inherit dpkg
+
+DESCRIPTION = "Secure Boot OTP key provisioning tool"
+
+DEBIAN_BUILD_DEPENDS = "openssl, u-boot-tools, device-tree-compiler"
+
+SRC_URI = " \
+    file://its \
+    file://keys \
+    file://make-otpcmd.sh \
+    file://rules"
+
+do_prepare_build[cleandirs] += "${S}/debian"
+do_prepare_build() {
+    deb_debianize
+    cd ${S}
+    cp -rPf ../its ${S}/its
+    ln -sf ../keys ${S}/keys
+    ln -sf ../make-otpcmd.sh ${S}
+    echo "otpcmd.bin /usr/lib/secure-boot-otp-provisioning/" > \
+            ${S}/debian/secure-boot-otp-provisioning.install
+}
+
+OTPCMD_MODE ?= "provision"
+OTP_MPK ?= "./keys/custMpk.pem"
+OTP_SMPK ?= "./keys/custSmpk.pem"
+OTP_BMPK ?= ""
```

Comment:
custBmpk.* thus appear to remain unused keys.

Reply:
Unused by default, however could be overwritten by yaml option file.

Check the README.md (under this folder):

```
### Provision the additional BMPK

If the BMPK need to be programmed together, before building the firmware, link
the `key-provision.its` to `key-provision-3keys.its.template`:


> # cd to files/its/
> ln -sf key-provision-3keys.its.template key-provision.its

then uncomment below line in `meta-iot2050/kas/opt/key-provision.yml`:

> # OTP_BMPK ?= "./keys/custBmpk.pem"

```